synapse/storage/databases/main/event_push_actions.py

```diff
@@ -615,7 +635,10 @@ def get_no_receipt(
                 stream_ordering=row[2],
                 actions=_deserialize_action(row[3], row[4]),
             )
-            for row in after_read_receipt + no_read_receipt
+            for row in push_actions
```

Comment:
to help with readability, especially since you're adding a condition, I'd be tempted to destructure this tuple and then refer to everything by name:

```suggestion
            for event_id, room_id, stream_ordering, actions, highlight in push_actions
```

Reply:
https://github.com/matrix-org/synapse/pull/13597/commits/b7a66e330afa97d686a0f38e12caaf1308f583d6